collector/package.json

```diff
@@ -34,7 +32,6 @@
     "mime": "^3.0.0",
     "minimatch": "5.1.0",
     "moment": "^2.29.4",
-    "multer": "^1.4.5-lts.1",
```

Comment:
Multer is only used for uploading, so we dont need this in collector

Reply:
https://github.com/Mintplex-Labs/anything-llm/commit/719521c307d81c430d7d90872768d6de556c38d7